hack/make-rules/go-test/unit.sh

```diff
@@ -64,5 +64,7 @@ fi
   umask 0022
   mkdir -p "${JUNIT_RESULT_DIR}"
   "${REPO_ROOT}/_bin/gotestsum" --junitfile="${JUNIT_RESULT_DIR}/junit-unit.xml" \
-    -- "./${folder_to_test}"
+    -- \
+    ${PROW_UNIT_TEST_EXTRA_FLAGS[@]} \
```

Comment:
Will this cause an error when unset?

Reply:
Hmm, I think this depends on the Bash version. I think the portable syntax is https://stackoverflow.com/a/61551944/437583. Will update.

Reply:
Updated.